Gopkg.toml

```diff
@@ -19,6 +19,10 @@
   name = "github.com/prometheus/prometheus"
   branch = "master"
 
+[[constraint]]
```

Comment:
Didn't we want to have comments for newly added constraints?

Reply:
good spot